src/server/svr_func.c

```diff
@@ -3713,11 +3704,6 @@ set_entity_resc_sum_queued(job *pjob, pbs_queue *pque, attribute *altered_resc,
 	if (rc_final)
 		return rc_final;
 
-	if (op == INCR)
-		pjob->ji_entity_limit_set = 1;
-	else
-		pjob->ji_entity_limit_set = 0;
-
```

Comment:
removed `pjob->ji_entity_limit_set` as it created phantom usage when multiple queued limits are in force.